packages/table/src/Table.story.jsx

```diff
@@ -0,0 +1,300 @@
+/* eslint-disable */
+import React from 'react';
+import { Table, Row, Cell, TableHeader, HeaderRow, DataType } from '.';
+import { defaultData, multiRowData } from './fixtures';
+
+export default {
+  title: 'Packages/Table',
+  component: Table,
+  args: {
+    data: defaultData,
+    withHeaders: {
+      type: 'boolean',
+    },
+  },
+  parameters: {
+    controls: { exclude: ['children', 'data', 'columns'] },
+  },
+};
+
+const Template = ({ children, withHeaders, ...args }) => (
+  <Table {...args}>{children(withHeaders)}</Table>
+);
+
+export const Basic = Template.bind({});
+Basic.args = {
+  withHeaders: false,
+  data: defaultData.slice(0, 8),
+  columns:
+    <HeaderRow>
+        <TableHeader key="name" label="Name" dataType="string" />
+        <TableHeader key="age" label="Age" dataType="number" />
+        <TableHeader key="color" label="Color" dataType="string" />
+        <TableHeader key="location" label="Location" />
+      </HeaderRow>,
+  children:
+    withHeaders =>
+    ({ datum }) =>
+      (
+        <Row key={datum.name}>
+          <Cell isHeader={withHeaders}>{datum.name}</Cell>
+          <Cell>{datum.age}</Cell>
+          <Cell>{datum.color}</Cell>
+          <Cell>{datum.location}</Cell>
+        </Row>
+      ),
+}
+
+export const BuiltInZebraStripes = Template.bind({});
+BuiltInZebraStripes.args = {
+  withHeaders: false,
+  columns:
+    <HeaderRow>
+        <TableHeader key="name" label="Name" dataType="string" />
+        <TableHeader key="age" label="Age" dataType="number" />
+        <TableHeader key="color" label="Color" dataType="string" />
+        <TableHeader key="location" label="Location" />
+      </HeaderRow>,
+  children:
+    withHeaders =>
+    ({ datum }) =>
+      (
+        <Row key={datum.name}>
+          <Cell isHeader={withHeaders}>{datum.name}</Cell>
+          <Cell>{datum.age}</Cell>
+          <Cell>{datum.color}</Cell>
+          <Cell>{datum.location}</Cell>
+        </Row>
+      ),
+}
+
+export const CustomLogic = Template.bind({});
+CustomLogic.args = {
```

Comment:
There's some weird stuff going on in this story 
<img width="1564" alt="Screen Shot 2022-04-28 at 5 09 05 PM" src="https://user-images.githubusercontent.com/2414030/165846450-1c270c41-317d-4657-809d-9ad23a78963c.png">